python/vendor/fb303/FacebookService-remote

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
```

Comment:
What is the license of this file? It is probably the license of fb303, since it was auto-generated from the fb303.thrift file. We should find out what the license is and update LICENSE and NOTICE.

Reply:
Okay, it looks like this is from an ASF repository and the thrift source is licensed with ALv2, so that's good. We should just need to state in the LICENSE file that this is from Thrift (and that the Hive one is from Hive) and is licensed under ALv2. Thrift has a [boilerplate NOTICE](https://github.com/apache/thrift/blob/7a5f29d48d6b99f14190d9d7bff0e63170965087/NOTICE), so there's nothing to add to our NOTICE. [Hive's NOTICE](https://github.com/apache/hive/blob/714c260e4a7c6b147c897718a33e693699267792/NOTICE) also doesn't contain anything that would require updates to Iceberg's NOTICE.

Reply:
Thanks for digging into this. I've added both of them to the `LICENSE` file.